source/includes/_commands.md

```diff
@@ -127,59 +125,61 @@ message AddSignatory {
 
 Field | Description | Constraint
 -------------- | -------------- | --------------
-Account ID | Account in which to add signatory (username@domain) | username@domain
+Account ID | Account to which to add signatory | account_name@domain
 Public key | Signatory to add to account | ed25519 public key
 
 ### Validation
 
 Two cases:
-Case 1. When transaction creator wants to add signatory to their account and he or she has permission CanAddSignatory
+Case 1. Transaction creator wants to add signatory to own account, having permission CanAddSignatory
 Case 2. Transaction creator was granted with CanAddSignatory permission to add signatory to this account
 
-## Create asset
+<aside class="notice">
+Granting specific rights in the system allows other account to perform actions over the account, which granted such rights. They can be revoked if needed, and for more information — please check <a href="#permission-model">permission model</a> section.
+</aside>
+
+## Append role
 
 ### Purpose
 
-Purpose of _сreate asset_ is to create new type of asset, specific for a domain.
+The purpose of _append role_ is to promote an account to some created role in the system, where role is a set of permissions account has to perform an action (command or query).
 
 ### Structure
 
 ```protobuf
-message CreateAsset {
-    string asset_name = 1;
-    string domain_id = 2;
-    uint32 precision = 3;
+message AppendRole {
+   string account_id = 1;
+   string role_name = 2;
 }
 ```
 ```json
 {
     "commands": [
         {
-            "command_type": "CreateAsset",
-            "asset_name": "usd",
-            "domain_id": "test",
-            "precision": "2"
+            "command_type": "AppendRole",
+            "account_id": "takemiya@test",
+            "role_name": "Administrator"
         }
     ], …
 }
 ```
 
 Field | Description | Constraint
 -------------- | -------------- | --------------
-Asset name | domain-unique name for asset | `[a-z]{1,9}`
-Domain ID | target domain to make relation with | should be created before the asset, `[a-z]{1,9}`
-Precision | number of digits after comma/dot | 0 <= precision <= uint32 max
+Account ID | id or account to append role to | already existent, account_name@domain
+Role name | name of already created role | already existent, `[A-Za-z0-9_]{1,7}`
 
 ### Validation
 
-1. Transaction creator has permission to create assets
-2. Asset name is valid and unique per domain
+1. Role should exist in the system
+2. Transaction creator should have permissions to append role (CanAppendRole)
+3. Account, which appends role, has set of permissions in his roles bigger than or equal to the size of permission set of appended role.
 
 ## Create account
 
 ### Purpose
 
-Purpose of _create account_ is to make new entity similar to wallet, which stores assets and identities.
+The purpose of _create account_ is to make entity in the system, capable of sending transactions or queries, storing signatories, personal data and identifiers.
```

Comment:
capable of sending, or capable to send?

Reply:
https://en.oxforddictionaries.com/definition/capable looks like of + ing